CRM/Price/BAO/PriceSet.php

```diff
@@ -1513,4 +1513,16 @@ public static function flushPriceSets(): void {
     }
   }
 
+  /**
+   * Whitelist of possible values for the entity_table field in PriceSetEntity
+   *
+   * @return array
+   */
+  public static function entityTables(): array {
+    return [
+      'civicrm_contribution_page' => ts('Contribution Page'),
+      'civicrm_event' => ts('Event'),
```

Comment:
Re the format of this array, this will work but is the old format. The new format would look like this:
```suggestion
      [
        'id' => 'civicrm_contribution_page',
        'name' => 'ContributionPage,
        'label' => ts('Contribution Page'),
      ],
      'id' => 'civicrm_event',
      'name' => 'Event',
      'label' => ts('Event'),
    ],
```

Reply:
@mattwire you ok with making this change & then let's merge the PR?